image/qcow2/qcow2.go

```diff
@@ -893,6 +893,8 @@ func (img *Qcow2) readZero(p []byte, off int64) (int, error) {
 	return readZero(p, off, img.Header.Size)
 }
 
+var zeroBuffer = make([]byte, 64*1024)
```

Comment:
The length should vary depending on the read buf size?

Reply:
Buffer size is controlled by the caller so we cannot depend on it. I started with 64 which should be faster than smaller buffers, but the benchmark shows clearly that 4k is better.